databases/week1/main.sql

```diff
@@ -0,0 +1,127 @@
+----1. Find out how many tasks are in the task table
+SELECT
+    count(id)
```

Comment:
COUNT(id) as task_amount
_Pointer_ 
1) COUNT is a SQL keyword so all caps
2) Columns created by you have to be named